mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
I find delta as posted by me above far more understandable than having `.value` in some paths and not in others. 

If we want everything uniform then all the leaf values should be under `value` key in the schema, so the lat long object would go under `navigation.position.value` and not directly under `navigation.position`. But that is another issue.